src/go/cmd/pkg.go

```diff
@@ -46,8 +49,9 @@ type Pkg struct {
 
 // NewPkg loads the package in the specified directory.
 // It's required there is only one package in the directory.
-func NewPkg(dir, moduleName string) (*Pkg, error) {
+func NewPkg(dir, moduleName, modulePath string) (*Pkg, error) {
```

Comment:
Doesn't `moduleName` become duplicate now that we're specifying the full module path (which I assume is really the module identity)?

Reply:
Removed. You also remind me to fix the possible problem for v2+ version packages.